setup.py

```diff
@@ -8,7 +8,7 @@
 
 setup(
     name='tzwhere',
```

Comment:
Consider yelp-tzwhere so internal packges names will never clash with future official pytzwhere versions.  Leave out of PRs you upstream, ofcourse.